src/compas_blender/artists/meshartist.py

```diff
@@ -166,16 +166,29 @@ def draw(self,
              vertices: Optional[List[int]] = None,
              edges: Optional[List[Tuple[int, int]]] = None,
              faces: Optional[List[int]] = None,
-             vertexcolor: Optional[Union[str, Color, List[Color], Dict[int, Color]]] = None,
-             edgecolor: Optional[Union[str, Color, List[Color], Dict[int, Color]]] = None,
-             facecolor: Optional[Union[str, Color, List[Color], Dict[int, Color]]] = None
+             vertexcolor: Optional[Union[str, RGBColor, List[RGBColor], Dict[int, RGBColor]]] = None,
```

Comment:
perhaps also here we should stop allowing for colors to be specified in hex format